inc/functions/options.php

```diff
@@ -61,18 +67,16 @@ function rocket_is_plugin_active( $plugin ) {
  * @source wp-admin/includes/plugin.php
  *
  * @param string $plugin Plugin folder/main file.
+ *
+ * @return bool true if multisite and plugin is active for network; else, false.
  */
 function rocket_is_plugin_active_for_network( $plugin ) {
 	if ( ! is_multisite() ) {
 		return false;
 	}
 
 	$plugins = get_site_option( 'active_sitewide_plugins' );
-	if ( isset( $plugins[ $plugin ] ) ) {
-		return true;
-	}
-
-	return false;
+	return ( isset( $plugins[ $plugin ] ) );
```

Comment:
are the wrapping parenthesis required?

Reply:
They are not. I have a habit of adding them to make it clear that it's a conditional expression. But definitely not required.

What do you think @Tabrisrp? Do you find them unnecessary?

Reply:
I understand, we have to be sure to be consistent with this though if we start doing that. We need to agree as a team with that practice and everyone will be doing it.

Reply:
This one I think is too much. I removed it https://github.com/wp-media/wp-rocket/pull/2483/commits/befc799b10763ff51e6d5c80e2f0839aa8896434